### PR TITLE
LPS-148665 Create unit tests for openConfirmModal and openAlertModal

### PR DESCRIPTION
**Pull request by diegonvs**
## What it does

Create jest tests for validating the values passed to openModal

## Considerations

I needed to create separated files for openConfirmModal and openAlertModal because I was getting issues when mocking openModal

More ctx: https://liferay.slack.com/archives/C3JBR21HA/p1664995465348219?thread_ts=1664995008.619039&cid=C3JBR21HA

### CODE REVIEW

**Comment by diegonvs** [comment]
Currently, I'm working on tests for Modal component which is used on openModal/open*Modal

**Comment by diegonvs** [comment]
>  I'm working on tests

They're mostly covered on ClayModal tests on clay repo.

**Comment by diegonvs** [comment]
ci:test:sf

**Review by bryceosterhaus — APPROVED** [review]
Looks good!

File: modules/apps/frontend-js/frontend-js-web/test/liferay/modal/Modal.js [hunk 1]
```diff
@@ -12,11 +12,27 @@
...
+
+jest.mock(
+	'../../../src/main/resources/META-INF/resources/liferay/modal/Modal',
+	() => ({
+		...jest.requireActual(
+			'../../../src/main/resources/META-INF/resources/liferay/modal/Modal'
+		),
+		openModal: jest.fn((args) => args),
+	})
+);
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-web/test/liferay/modal/Modal.js:35]: What is the intent with this code? :thinking:

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js [hunk 2]
```diff
@@ -190,9 +190,9 @@ Liferay.Util.addParams = addParams;
 
 Liferay.Util.openAlertModal = (...args) => {
 	Liferay.Loader.require(
-		'frontend-js-web/liferay/modal/Modal',
+		'frontend-js-web/liferay/modal/commands/OpenAlertModal',
 		(commands) => {
-			commands.openAlertModal(...args);
+			commands.default(...args);
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:195]: Direct imports will still break, as they rely on [module export](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js#L60-L65).

**Review by markocikos — CHANGES_REQUESTED** [review]
@diegonvs I think it's a good idea to extract utilities from `Modal.js` into separate files. But:
- We should keep the naming pattern of util files in `frontend-js-web`. Names should be `open_*_modal.js`
- Path and name of test files should be the same as source, `/test/liferay/modal/commands/open_*_modal.js`

This may simplify code in the new tests, as we probably won't need the awkward partial mock.

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
LGTM!

**Comment by markocikos** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:forward

**Comment by diegonvs** [comment]
ci:forward

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**